CONTRIBUTING.md

```diff
@@ -57,27 +81,38 @@ During markdown to HTML conversion, some predefined variables located in the [va
 For instance, the `varVersion` variable matches the current documentation version.
 To reference this variable, use the `${varVersion}` notation in _.md_ files.
 
+#### Markdown titles
+
+In markdown, lines starting with `#` are converted to `<hx>` tags where _x_ is the number of `#` : 
+* use `#` for `h1` tag
+* use `##` for `h2` tag
+* use `###` for `h3` tag
+* etc... up to `h6`
+
+Checkout [the markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes) for more information.
```

Comment:
Can we link to https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers or https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet instead?